arcgis-ios-sdk-samples/Layers/Display dimensions/DisplayDimensions.storyboard

```diff
@@ -0,0 +1,147 @@
+                        <color key="backgroundColor" systemColor="tertiarySystemBackgroundColor"/>
+                        <constraints>
+                            <constraint firstItem="z8S-xe-km7" firstAttribute="leading" secondItem="5EZ-qb-Rvc" secondAttribute="leading" id="8xv-uT-P4z"/>
+                            <constraint firstItem="z8S-xe-km7" firstAttribute="bottom" secondItem="Sgq-PD-jL8" secondAttribute="top" id="Xxy-N9-Kvz"/>
+                            <constraint firstItem="Sgq-PD-jL8" firstAttribute="leading" secondItem="5EZ-qb-Rvc" secondAttribute="leading" id="bZS-qh-CEh"/>
+                            <constraint firstItem="Sgq-PD-jL8" firstAttribute="bottom" secondItem="vDu-zF-Fre" secondAttribute="bottom" id="k3j-Xz-sco"/>
+                            <constraint firstItem="Sgq-PD-jL8" firstAttribute="trailing" secondItem="5EZ-qb-Rvc" secondAttribute="trailing" id="k4X-yL-YU8"/>
+                            <constraint firstItem="z8S-xe-km7" firstAttribute="top" secondItem="vDu-zF-Fre" secondAttribute="top" id="naM-7d-1KV"/>
+                            <constraint firstItem="z8S-xe-km7" firstAttribute="trailing" secondItem="5EZ-qb-Rvc" secondAttribute="trailing" id="uIX-M5-eZt"/>
+                        </constraints>
+                    </view>
+                    <connections>
+                        <outlet property="mapView" destination="z8S-xe-km7" id="rpg-tc-eJu"/>
+                        <outlet property="settingsBarButtonItem" destination="qDJ-04-mpc" id="z6I-H6-dby"/>
+                    </connections>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="Ief-a0-LHa" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="139" y="84"/>
+        </scene>
+        <!--Display Dimensions Settings View Controller-->
+        <scene sceneID="Ocp-fF-Pp2">
+            <objects>
+                <tableViewController id="q8d-Qn-axY" customClass="DisplayDimensionsSettingsViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="static" style="grouped" separatorStyle="default" allowsSelection="NO" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="18" estimatedSectionHeaderHeight="-1" sectionFooterHeight="18" estimatedSectionFooterHeight="-1" id="wBA-eU-15k">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="842"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <color key="backgroundColor" red="0.94901960784313721" green="0.94901960784313721" blue="0.96862745098039216" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
```

Comment:
I think this is an Apple bug with the storyboard. If I choose the Grouped style, the background color will be set to System Background Color. When I change it to Default, instead of having a nil as the key of color, it will use this custom color.